src/config/schema.types.d.ts

```diff
@@ -65,15 +65,8 @@ type YamlSchema = {
     phone: string
   }[]
   test_accounts: {
-    ref: string
     phone: string
     code: string
-    needUsdWallet?: boolean
```

Comment:
probably will require config updates in staging and prod

Reply:
Good point 

But I think we only use it with phone + code. 

I'll have a look. 

Ajv would let us know if that's not the case and not let the pod start

Reply:
https://github.com/GaloyMoney/galoy-deployments/pull/4448